app/javascript/components/recipes/new/add-button.vue

```diff
@@ -0,0 +1,22 @@
+ <template> 
+  <button
+      @click="clickButton"
+  > + </button>
+</template>
+
+<script>
+export default {
+  props: {
+    object: {type: Object, required:true}
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  A space is required before '}'                                                    object-curly-spacing